stl/src/multprec.cpp

```diff
@@ -63,16 +63,16 @@ void __CLRCALL_PURE_OR_CDECL _MP_Mul(
 
     // Knuth, vol. 2, p. 268, Algorithm M
     // M1: [Initialize.]
-    for (int i = 0; i < m + n + 1; ++i) {
+    for (unsigned int i = 0; i < m + n + 1; ++i) {

```

Comment:
The correctness of the `int` => `unsigned int` change seems reasonably easy to verify, but I don't believe that this is a clear improvement, and @fsb4000 has found potential negative codegen impact. Although I have a slight preference for unsigned types in general, I don't see sufficient value in changing this existing code, so I believe these changes should be dropped.